src/octoprint/printer/standard.py

```diff
@@ -1744,12 +1744,19 @@ def on_comm_print_job_paused(self, suppress_script=False, user=None):
         )
         if payload:
             eventManager().fire(Events.PRINT_PAUSED, payload)
+            if "position" in payload:
+                position = str(payload["position"])
+            else:
+                position = "unknown"
+            fileposition = str(self._comm.getFilePosition())
```

Comment:
`self._comm` might be `None` under certain (very unlikely but still possible) circumstances, so this needs a null-check.